Sources/MFBActionBinding.m

```diff
@@ -0,0 +1,227 @@
+//
+//  MFBActionBinding.m
+//  Pods
+//
+//  Created by Nickolay Tarbayev on 28.07.2017.
+//
+//
+
+#import <objc/message.h>
+
+#import "MFBActionBinding.h"
+#import "MFBBindingInfo.h"
+#import "MFBBinding.h"
+
+static void *ObservableValueChange = &ObservableValueChange;
+
+static NSString *const TriggeringBindingsGroup = @"TriggeringBindings";
+static NSString *const ActionBindingsGroup = @"ActionBindings";
+
+@interface MFBActionBinding () <MFBUnbindable>
+@property (nonatomic) SEL actionSelector;
+@end
+
+@implementation MFBActionBinding {
+    BOOL _isBound;
```

Comment:
These two flags could be replaced with one enum of `NotBound`, `Bound`, `BoundAndUpdating`. This rules out invalid states like "not bound, but updating" statically.